resources/com.sap.piper/templates/whitesourceVulnerabilities.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>${reportTitle}</title>
+    <style type="text/css">${style}</style>
+</head>
+<body>
+    <h1>${reportTitle}</h1>
+    <h2>
+        <span>
+            <i>WhiteSource product name: ${whitesourceProductName}</i><br />
+            <i>Filtered project names: ${whitesourceProjectNames?:''}</i>
+        </span>
+    </h2>
+    <div>
+        <h3> total number of vulnerabilities: ${totalVulnerabilities}<br />
+        total number of high/critical vulnerabilities with CVSS score >= ${cvssSeverityLimit}: ${totalSevereVulnerabilities}
+        </h3>
+    </div>
+    <p>Snapshot taken:${now} GMT</p>
```

Comment:
Nitpick: `UTC` is the technically correct term

Reply:
Great spot actually! Ugly leftover and in deed putting potentially wrong assumptions. Now it's explicitly converted to UTC.